src/main/resources/org/synyx/urlaubsverwaltung/core/mail/notify_expired_remaining_vacation_days.ftl

```diff
@@ -0,0 +1,3 @@
+Hallo ${recipientNiceName},
+
+leider ist dein Resturlaub zum 01.04.${year?string.computer} in Höhe von ${expiredRemainingVacationDays} Tagen verfallen.
```

Comment:
Paar Gedanken:
* Links in die UV anbieten
* Übersicht über den aktuellen Anspruch, nach dem Invalidieren, in der Mail? Sodass man direkt ein vorher/nachher sieht?
* Datum lieber komplett reingeben?